app/gui2/src/components/ExtendedMenu.vue

```diff
@@ -1,46 +1,49 @@
 <script setup lang="ts">
 import SvgIcon from '@/components/SvgIcon.vue'
+import { isMacLike } from '@/composables/events'
 import { ref } from 'vue'
 
 const isDropdownOpen = ref(false)
 
 const props = defineProps<{
   zoomLevel: number
 }>()
-const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: [] }>()
+const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: []; toggleCodeEditor: [] }>()
+
+const toggleCodeEditorShortcut = isMacLike ? 'Cmd + `' : 'Ctrl + `'
```

Comment:
Instead of hardcoding this shortcut string, let's add a new method to the object returned from `defineKeybinds`, so that we can read the shortcuts by action name. Then this could be something like `codeEditorBindings.displayBinding('toggle')`.

Reply:
AFAIK, it's not so easy to add the function, as it parses the string and changes the representation.

@vitvakatu If refactoring defineKeybinds would take more than a day, let's make a follow-up issue and add TODO with link here.

When implemented, the function returning keybinds should be also used in area selection unit tests.

Reply:
I added the API itself, but I want it to use type information (knowing the list of shortcuts which are actually set). Added a TODO for now, to not block this PR.